parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -47,7 +47,9 @@ use sp_version::NativeVersion;
 use sp_version::RuntimeVersion;
 
 use codec::{Decode, Encode, MaxEncodedLen};
-use constants::{consensus::*, currency::*, fee::WeightToFee};
+#[cfg(feature = "parameterized-consensus-hook")]
```

Comment:
we don't need the feature flag in system-parachain runtimes either, as they're binaries, not libraries.